packages/umi-bundle-defaults/src/index.ts

```diff
@@ -1,10 +1,26 @@
 import { Umi, createUmi as baseCreateUmi } from '@metaplex-foundation/umi';
+import type { ChunkGetAccountsRpcOptions } from '@metaplex-foundation/umi-rpc-chunk-get-accounts';
 import type { Web3JsRpcOptions } from '@metaplex-foundation/umi-rpc-web3js';
+import type { Connection as Web3JsConnection } from '@solana/web3.js';
 import { defaultPlugins } from './plugin';
 
-export const createUmi = (
+export function createUmi(
   endpoint: string,
-  rpcOptions?: Web3JsRpcOptions
-): Umi => baseCreateUmi().use(defaultPlugins(endpoint, rpcOptions));
+  rpcOptions?: Web3JsRpcOptions & ChunkGetAccountsRpcOptions
+): Umi;
+export function createUmi(
+  connection: Web3JsConnection,
+  rpcOptions?: ChunkGetAccountsRpcOptions
+): Umi;
+export function createUmi(
+  endpointOrConnection: string | Web3JsConnection,
+  rpcOptions?: Web3JsRpcOptions & ChunkGetAccountsRpcOptions
+): Umi {
+  return baseCreateUmi().use(
+    typeof endpointOrConnection === 'string'
+      ? defaultPlugins(endpointOrConnection, rpcOptions)
+      : defaultPlugins(endpointOrConnection, rpcOptions)
```

Comment:
this is slightly weird since its the same function call params, but ts doesnt like it without the ternary

![image](https://github.com/metaplex-foundation/umi/assets/93682696/740f49c5-bad1-4877-9ca2-7fd6b4876c66)